docker-for-windows/release-notes.md

```diff
@@ -11,10 +11,32 @@ toc_max: 2
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 3.3.2
+2021-04-29
+
+> [Download](?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64)
```

Comment:
The latest download link can be copied from the previous release and has no build number.

```suggestion
> [Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64)
```

Reply:
Sorry, you were too fast before I added this suggestion :-)